core/pom.xml

```diff
@@ -37,6 +38,11 @@
         <test.exclude>None</test.exclude>
         <test.include>%regex[.*[^o].class]</test.include><!-- exclude module-info.class-->
         <web.console.version>0.3.3</web.console.version>
+        <qdb.path>rust/qdbr</qdb.path>
+        <!--
+        For production builds, use -P qdb-release to compile optimized Rust binaries.
+        -->
+        <qdb.release>true</qdb.release>
```

Comment:
issue: This should be defaulted to `false` not `true`, or we never get a debug binary. Debug binaries are also faster to build during regular development.

Reply:
I've also noticed the `qdb-release` profile is missing and should be added.

Reply:
The profile is also mising, it should either be added or the comment should be updated to mention `.. use -Dqdb.release=true to compile ..`.

Reply:
fixed in https://github.com/questdb/questdb/pull/4439/commits/5c962d386513b74eefe07e95a2bf630270d30b3e

Ant has separate tasks for release and non-release as required

![image](https://github.com/questdb/questdb/assets/7276403/f6efc4a0-eac2-4079-89b6-b1ed88bfc58b)